lua/telescope/_extensions/neorg/insert_link.lua

```diff
@@ -135,20 +135,20 @@ return function(opts)
 
                 vim.api.nvim_put(
                     {
-                        "["
-                            .. entry.ordinal:gsub(":$", "")
-                            .. "]"
-                            .. "("
+                        "{"
                             .. inserted_file
-                            .. entry.display:gsub("^(%W+)%s+.+", "%1")
+                            .. entry.display:gsub("^(%W+)%s+.+", "%1 ")
                             .. entry.ordinal:gsub("[%*#%|_]", "\\%1")
```

Comment:
If I'm understanding this regex correctly this is checking for the various symbols that need escaping. If that is true, then we need to add all detached modifiers which can be linked against, right @vhyrro?

Reply:
Honestly I don't think we need to escape custom symbols anymore, only `}` symbols in the case of `{this}` and `]` symbols in the case of `[this]`.